doc/source/tune/_tutorials/tune-xgboost.rst

```diff
@@ -501,4 +508,15 @@ to tune the hyperparameters to yield the best results. In our simple example,
 Tuning the parameters didn't make a huge difference for the accuracy.
 But in larger applications, intelligent hyperparameter tuning can make the
 difference between a model that doesn't seem to learn at all, and a model
-that outperforms all the other ones.
\ No newline at end of file
+that outperforms all the other ones.
+
+Further References
+------------------
+
+(Need to format the below into links)
+XGBoost Hyperparameter Tuning - A Visual Guide https://kevinvecmanis.io/machine%20learning/hyperparameter%20tuning/dataviz/python/2019/05/11/XGBoost-Tuning-Visual-Guide.html
```

Comment:
Hey @krfricke can you fix this up? 

I think they should be in a format like `Title [link]`.

thanks!